e2e/openshift/openshift_packaging_test.go

```diff
@@ -233,17 +233,17 @@ var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
 
 // randSeq returns a string of n letters
 func randSeq(n int) string {
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
 	b := make([]rune, n)
 	for i := range b {
-		b[i] = letters[rand.Intn(len(letters))]
+		b[i] = letters[r.Intn(len(letters))]
 	}
 	return string(b)
 }
 
 // wriiteFile writes out b byte array to a random file in the
 // os.TempDir and returns the name of that file.
 func writeFile(t *testing.T, b []byte) string {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
This just wasn't needed?

Reply:
rand.Seed() is deprecated in Go 1.20. This is the new way to generate random strings as per Go 1.20.

Lint is failing saying `rand.Seed()` is deprecated, hence this change is necessary.

Reply:
Lint job failure without this change: https://github.com/cockroachdb/cockroach-operator/actions/runs/4728824923/jobs/8390733554

Reply:
We have moved the alternative function call to `randSeq` function where we generate the random string. Earlier rand.Seed initialise the default resource once for random method. Now you generate the `rand.Rand` with NewSource function in `randSeq` method itself and use that Rand object to generate the random string.